pkgs/flox/default.nix

```diff
@@ -97,15 +99,24 @@ in
           --bash <($out/bin/flox --bpaf-complete-style-bash) \
           --fish <($out/bin/flox --bpaf-complete-style-fish) \
           --zsh <($out/bin/flox --bpaf-complete-style-zsh)
+
+        wrapProgram "$out/bin/flox"                                       \
+                    --prefix PATH :                                       \
+                      "${flox-bash}/bin:${flox-bash}/libexec:${git}/bin"
```

Comment:
Won't this inject those into the path for anything that flox invokes? It is only nix that needs git in its path, so flox should be invoking a wrapped version of nix.

It's essential we don't modify PATH (except for the purpose of adding environment bin directories) when running `flox activate` or `flox run`, for instance.